tests/chmod/11.t

```diff
@@ -63,6 +63,12 @@ for type in regular fifo block char socket symlink; do
 		create_file ${type} ${n1} 0640 65534 65534
 		expect 0 symlink ${n1} ${n2}
 		case "${os}" in
+		Darwin)
```

Comment:
Yes, but there aren't any guarantees that this will remain that way >_>... Just one step to making OS X work with pjdfstest.